dance/datasets/singlemodality.py

```diff
@@ -52,12 +52,12 @@ class ScDeepSortDataset(BaseDataset):
         "test_mouse_Kidney203_data.csv":        "https://www.dropbox.com/s/kmos1ceubumgmpj?dl=1",
     }  # yapf: disable
 
-    def __init__(self, full_download=False, train_dataset=None, test_dataset=None, species=None, tissue=None,
+    def __init__(self, full_download=False, dataset=None, test_dataset=None, species=None, tissue=None,
                  train_dir="train", test_dir="test", map_path="map", data_dir="./"):
         super().__init__(data_dir, full_download)
 
         self.data_dir = data_dir
-        self.train_dataset = train_dataset
+        self.dataset = dataset
         self.test_dataset = test_dataset
```

Comment:
Is `test_dataset` ever used in the updated imputation dataset setting? If not, we should remove it from the `ImputationDataset` object.